pkg/smartcontract/rpcbinding/binding.go

```diff
@@ -413,6 +413,10 @@ func Generate(cfg binding.Config) error {
 				mfst.ABI.Methods = dropStdMethods(mfst.ABI.Methods, standard.Nep11NonDivisible)
 				ctr.IsNep11ND = true
 			}
+			if standard.ComplyABI(cfg.Manifest, standard.Nep24Royalty) == nil {
+				mfst.ABI.Methods = dropStdMethods(mfst.ABI.Methods, standard.Nep24Royalty)
+				ctr.IsNep11ND = true
```

Comment:
> ctr.IsNep11ND = true

It's not.

Reply:
And you need to fix RPC bindings generator. Based on `ctr.IsNep24` flag RPC bindings generator must include package `nep24` to the generated contract reader.